cli/src/term.rs

```diff
@@ -228,8 +229,15 @@ impl Reporter for SpinnerReporter {
         self.solc_io_report.log_compiler_input(input);
     }
 
-    fn on_solc_success(&self, _solc: &Solc, _version: &Version, output: &CompilerOutput) {
+    fn on_solc_success(
+        &self,
+        _solc: &Solc,
+        _version: &Version,
+        output: &CompilerOutput,
+        duration: &Duration,
+    ) {
         self.solc_io_report.log_compiler_output(output);
+        self.send_msg(format!("Finished in {:.2?}", duration));
```

Comment:
Should we make it clearer that this is just the time `solc` took to compile, like "Solc finished in", perhaps?